app.py

```diff
@@ -26,7 +30,12 @@
 def post_random_game_from_users_collection() -> Union[str, Response]:
     args = request.args
     user = args.get("user")
-    bgg_companion_api = BggCompanionApi(request_client=RequestsRetryClient())
+
+    # BGG's API usually responds with a 202 and retrying gets the 200
+    retry_codes = list(DEFAULT_HTTP_RETRY_CODES) + [202]
```

Comment:
is there a reason why you only want to add this logic to this one view? would it make more sense to add `202` to the `DEFAULT_HTTP_RETRY_CODES` list in the `src.utils.requests_retry_client` module? that way you can get this behavior every time you call the BGG API

Reply:
Yeah so my thought process was since 202 might be a weird HTTP code to retry and is more specific to the bgg API, maybe the responsibility of BGG's weird retry needs should be held when the class is instantiated?  Although right now bgg API is the only API here using the retry client.... this sort of keeps in separate incase another API being called is introduced?

Reply:
i see what you mean, perhaps you could have a separate `BGGRequestsRetryClient` that includes that status code in its retry codes list? from my perspective it feels a little weird that users of the `RequestsRetryClient` need to know about the internals of the class (like how the retry logic is implemented and what extra error codes to retry on) so i suggest moving that logic out of the callers and into the class somehow

Reply:
Ahhhhh yeah that totally makes sense.  That it feels weird there's some very detailed specifics callers of RequestsRetryClient need to know in order to call it with special codes.  I wasn't thinking about it from that lends but definitely agree!

Reply:
@nickdibari After thinking about it more I think latest commit makes the most sense.  Simply a new list in the retry client called BGG_API_HTTP_RETRY_CODES.  I feel like this accomplishes both : keeps separation of where BGG specific codes belong AND keep the details of the class away from the caller so they aren't responsbile for knowing.